packages/template-retail-react-app/app/pages/check-email/index.jsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {useLocation} from 'react-router-dom'
+import {Box, Container} from '@salesforce/retail-react-app/app/components/shared/ui'
+import Seo from '@salesforce/retail-react-app/app/components/seo'
+import CheckEmail from '@salesforce/retail-react-app/app/components/check-email'
+
+const CheckEmailPage = () => {
+    const location = useLocation()
+    console.log('location.state = ' + location.state)
+    const email = ''
+
+    return (
+        <Box data-testid="check-email-page" bg="gray.50" py={[8, 16]}>
+            <Seo title="Check your email" description="Check your email" />
```

Comment:
Do we translate these on other pages? If we don't then please ignore this comment.